package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "customerio-node",
   "description": "A node client for the Customer.io event API. http://customer.io",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
Generally we do separate version + release PRs. For now, this is ok to leave in, but if another PR comes in before this is merged, then this will need to be reverted.

Reply:
additionally this is a  minor release, not a patch

Reply:
Done